lib/main_ltp.pm

```diff
@@ -20,6 +20,7 @@ use base 'Exporter';
 use Exporter;
 use testapi qw(check_var get_var);
 use autotest;
+use Archive::Tar;
```

Comment:
Do we need to use tar at `perl` level? There is a lot of shell scripting already, can't we use tar on sut with assert_script_run?

Reply:
I described this in the commit message :)
*Using it was required due AppArmor profile restrictions.* (many binaries was not allowed to use, like `mv` and `tar`).
Believe I tried before and I'm lazy enough to keep error prone shell code, but it was really needed. BTW this attempt required to use `system()` because `script_output()` failed for some uninitialized value (it's a test API, but `lib/main_ltp.pm` isn't running test yet).

Reply:
Wouldn't help if this logic was in `boot_ltp`? And test itself would dowload asset with similar logic like data file download.

Reply:
That's IMHO impossible. This code schedules number of tests (any row after `boot_ltp.pm`), so it'd be too late to instert new tests in `boot_ltp.pm`. But maybe I'm wrong, @mdoucha change  02dba8975500d69ee8ec272635bb89656afafb70, which allowed adding new tests on the fly would probably allowed it. But that's not updating the table during live view (you'd need to refresh page for each live view, now this is needed only when you set `INSTALL_LTP` for `ltp_*` tests).

Reply:
But I consider perl code less error prone than embedded shell, I actually prefer it.